src/realm/query_expression.hpp

```diff
@@ -2620,6 +2795,18 @@ class Columns : public Subexpr2<T> {
     }
 };
 
+template <class T>
+void Columns<T>::evaluate(size_t index, ValueBase& destination)
```

Comment:
There probably was a good reason at some point in my development history, but that seems to have disappeared.